base/src/main/java/com/smartdevicelink/streaming/video/VideoStreamingParameters.java

```diff
@@ -103,15 +92,16 @@ public VideoStreamingParameters(int displayDensity, int frameRate, int bitrate,
      * @param format
      * @param stableFrameRate
      */
+    @Deprecated
     public VideoStreamingParameters(int displayDensity, int frameRate, int bitrate, int interval,
-                                    ImageResolution resolution, VideoStreamingFormat format, boolean stableFrameRate){
+                                    ImageResolution resolution, VideoStreamingFormat format){
 	    this.displayDensity = displayDensity;
 	    this.frameRate = frameRate;
 	    this.bitrate = bitrate;
 	    this.interval = interval;
 	    this.resolution = resolution;
 	    this.format = format;
-	    this.stableFrameRate = stableFrameRate;
+	    this.stableFrameRate = true;
```

Comment:
Hello @shiniwat, I thank you for deprecating the old constructor but I eblieve we still need to include the new constructor so we should have both of the constructors listed here (The deprecated constructor will assign true for stableFrameRate)

```
@Deprecated
public VideoStreamingParameters(int displayDensity, int frameRate, int bitrate, int interval,
                                    ImageResolution resolution, VideoStreamingFormat format) {}

public VideoStreamingParameters(int displayDensity, int frameRate, int bitrate, int interval,
                                    ImageResolution resolution, VideoStreamingFormat format, boolean stableFrameRate) {}
```

Reply:
Understood. I have restored the constructor, and updated the method comments accordingly.